File: src/main/java/net/minestom/server/monitoring/TickMonitor.java
package net.minestom.server.monitoring;

public class TickMonitor {

    private final double tickTime;
    private final double acquisitionTime;

    public TickMonitor(double tickTime, double acquisitionTime) {
        this.tickTime = tickTime;
        this.acquisitionTime = acquisitionTime;
    }

    public double getTickTime() {
        return tickTime;
    }

    public double getAcquisitionTime() {
        return acquisitionTime;
    }
}

File: src/main/java/net/minestom/server/monitoring/TickMonitor.kt
package net.minestom.server.monitoring

import net.minestom.server.utils.validate.Check.stateCondition
import net.minestom.server.MinecraftServer.Companion.exceptionManager
import it.unimi.dsi.fastutil.longs.Long2LongOpenHashMap
import net.minestom.server.monitoring.ThreadResult
import java.util.concurrent.ConcurrentHashMap
import net.minestom.server.monitoring.BenchmarkManager
import java.lang.Runnable
import java.lang.InterruptedException
import net.minestom.server.MinecraftServer
import net.kyori.adventure.text.format.NamedTextColor

class TickMonitor(val tickTime: Double, val acquisitionTime: Double)